docs/streams/getting-started-streams.md

```diff
@@ -19,6 +19,8 @@ This page guides you through creating a stream, publishing messages to it, and s
 
 <Steps />
 
+**Note:** The code snippets in each step aren't the same as the code in the full demo file at the end.
```

Comment:
```suggestion
:::note
 The code snippets in each step aren't the same as the code in the full demo file at the end.
 :::
```

Reply:
This is the correct format for notes, cautions, tips, and warnings.

Reply:
Let's add a bit about why, though. Something like:

The step-by-step code snippets are designed to run iteratively, so they can build on each other and work even though the app is incomplete.

The full demo code is written as a complete app, with defined functions that are all called at the end, so more like a proper program.

Reply:
If you had time to turn the functions into steps, that would be cool. I understand that you probably do not. Maybe write a Jira ticket describing it so it doesn't get lost?